### ISSUE TITLE
Problem with kivy, old python3 version.

### ISSUE THREAD

**Issue by tct123** [qpython-android/qpython#206]
I have a problem with installing kivy (newest version) and upgrading pip.
When I am upgrading pip, it says that I need a newer python3 version.
When I install kivy it says that pip didnt found some dependencies.

**Comment by tct123** [comment]
?

**Comment by amstelchen** [comment]
QPython is a dead project.

**Comment by SukiCZ** [comment]
Pydroid 3 seems as a good alternative (using python 3.9)

**Comment by tct123** [comment]
Yes, but it is made by a russian company and it is closed source. I don't want to use pydroid 3 because of data protection and the current russian war of aggression.

**Comment by SukiCZ** [comment]
@tct123 Thanks. I didn't know that. Is there any other alternative you would recommend? 
My use case is to briefly check how Python behaves when I'm on mobile (no need to create difficult projects or use 3rd party packages).
